posixlib/src/main/resources/scala-native/sys/stat.c

```diff
@@ -42,9 +42,18 @@ void scalanative_stat_init(struct stat *stat,
     my_stat->st_uid = stat->st_uid;
     my_stat->st_gid = stat->st_gid;
     my_stat->st_size = stat->st_size;
-    my_stat->_st_atime = stat->st_atime;
-    my_stat->_st_mtime = stat->st_mtime;
-    my_stat->_st_ctime = stat->st_ctime;
+// see https://linux.die.net/man/2/stat
+#if defined(_BSD_SOURCE) || defined(_SVID_SOURCE) ||                           \
+    defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200809L ||                  \
+    defined(_XOPEN_SOURCE) && _XOPEN_SOURCE >= 700
+    my_stat->st_atim = stat->st_atim;
+    my_stat->st_mtim = stat->st_mtim;
+    my_stat->st_ctim = stat->st_ctim;
+#else
```

Comment:
A matter of stytle, not a requirement: an `#else // APPLE`
would make it easier to trace here.